src/MassTransit.SignalR.Tests/HubLifeTimeManagerTests.cs

```diff
@@ -194,9 +194,9 @@ public async Task WritingToGroupWithOneConnectionFailingSecondConnectionStillRec
                 var connection2 = HubConnectionContextUtils.Create(client2.Connection);
 
                 await manager.OnConnectedAsync(connection1).OrTimeout();
-                await manager.AddToGroupAsync(connection1.ConnectionId, "group");
+                await manager.AddToGroupAsync(connection1.ConnectionId, "group").OrTimeout();
```

Comment:
Should this be `.OrTimeout(TestTimeout)` ?

Reply:
yep, done